src/BMI/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Check for pcap_set_immediate_mode
+include(CheckSymbolExists)
+check_symbol_exists(pcap_set_immediate_mode "pcap.h" HAVE_PCAP_SET_IMMEDIATE_MODE)
+
+# Create bmi library
+add_library(bmi OBJECT
+  bmi_interface.c
+  bmi_port.c
+)
+
+# Set include directories
+target_include_directories(bmi PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}
+)
```

Comment:
Is this necessary? Is CMAKE_CURRENT_SOURCE_DIR not already searched for include?